### PR TITLE
Added attack feedback to flock structures

### PR DESCRIPTION
**Pull request by TobleroneSwordfish**
[FEATURE]
## About the PR 
Adds a shake animation when flock structures are hit with something and makes them use a crunchy glass sound when they are attacked with something heavy.

## Why's this needed? 
Attack feedback good, makes it obvious that this is a thing you can and should destroy.

### CODE REVIEW

**Comment by FlameArrow57** [comment]
Idk, do they need twitch feedback? I don't think there's anything else that really has twitch feedback, plus they have hit sound, attack particle, and chat message already

**Comment by TobleroneSwordfish** [comment]
It was feedback we got from the playtest, and players not knowing what they should be hitting is a longstanding problem with flock iirc

**Review by amylizzle — APPROVED** [review]

**Review by ZeWaka — APPROVED** [review]

**Comment by FlameArrow57** [comment]
Okay. Idk if this the right place to put the hit_twitch(), as cages already have a twitch thing on takeDamage, so hit twitch stuff would occur twice for them

**Review by stonepillars — APPROVED** [review]

File: code/obj/flock/cage.dm [hunk 1]
```diff
@@ -12,6 +12,7 @@
 	health_max = 30
 	alpha = 192
 	anchored = FALSE
+	hitTwitch = FALSE
```
Comment (FlameArrow57) [inline code/obj/flock/cage.dm:15]: I think a clarifying comment saying that it implements its own twitch stuff would be nice

File: code/obj/flock/structure/flock_structure_parent.dm [hunk 2]
```diff
@@ -17,6 +17,8 @@ ABSTRACT_TYPE(/obj/flock_structure)
 	var/health = 30
 	var/health_max = 30
 	var/bruteVuln = 1.2
+	///Should it twitch on being hit?
```
Comment (FlameArrow57) [inline code/obj/flock/structure/flock_structure_parent.dm:20]:
```suggestion
	///Should it twitch on being hit through attackby()?
```
Or something similar?

**Pull request merged**